src/modules/endpoint/errors/ParamMissingError.js

```diff
@@ -8,6 +10,14 @@ class ParamMissingError extends Error {
     super()
     this.message = this.getMessage(endpointName, paramName)
 
+    // https://github.com/babel/babel/issues/3083
```

Comment:
@Integreat/app-web This is needed for all es6 classes which extend an interrnal type of javascipt.